R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -564,6 +566,26 @@ model <- spark.logit(df, Species ~ ., regParam = 0.056)
 summary(model)
 ```
 
+#### Linear Support Vector Machine (SVM) Classifier
```

Comment:
this feels minor to me, but there was feedback that these section should be in the same order they are listed (ie sorted alphabetically)